homeassistant/components/http/auth.py

```diff
@@ -61,19 +63,19 @@ def async_sign_path(
 
     url = URL(path)
     now = dt_util.utcnow()
+    now_timestamp = int(now.timestamp())
+    expiration_timestamp = now_timestamp + int(expiration.total_seconds())
```

Comment:
JWT wants timestamps. Doing this here instead and passing to api_jws avoids all the expensive conversions and copies